src/VS.Web.CG.Mvc/Templates/Identity/Bootstrap3/Pages/Account/Manage/Account.Manage.ExternalLogins.cs.cshtml

```diff
@@ -62,7 +65,14 @@ namespace @thisNamespace
             OtherLogins = (await _signInManager.GetExternalAuthenticationSchemesAsync())
                 .Where(auth => CurrentLogins.All(ul => auth.Name != ul.LoginProvider))
                 .ToList();
-            ShowRemoveButton = user.PasswordHash != null || CurrentLogins.Count > 1;
+
+            string passwordHash = null;
+            if (_userStore is IUserPasswordStore<@(Model.UserClass)> userPasswordStore)
+            {
+                passwordHash = await userPasswordStore.GetPasswordHashAsync(user, HttpContext.RequestAborted);
+            }
+
```

Comment:
We don't need this check either because the stores we support in scaffolding are password stores